lib/client/api_test.go

```diff
@@ -500,3 +498,61 @@ func TestApplyProxySettings(t *testing.T) {
 		})
 	}
 }
+
+func TestParseSearchKeywords(t *testing.T) {
+	t.Parallel()
+
+	expected := [][]string{
```

Comment:
Can you make `expected` a field of the `testCases` anonymous struct?

When writing a new test it's nice to see the input and expected output together, rather than updating them in two separate places.

Reply:
sorry zac, i wasn't sure what you meant by `updating them in two places`? the expected values should not change? i put it out b/c they are the same expects for each tests that test with similar inputs but with different delimiters.

This is what it looks like if something does not match (which looks easy to understand, to me at least):
![image](https://user-images.githubusercontent.com/43280172/158245254-3817bf1b-78e7-4d74-ae2f-a5d1f7264f1e.png)

maybe my test is confusing? should i create individual tests for each delimiter?

Reply:
i separated the test to make it easier to understand: https://github.com/gravitational/teleport/pull/11016/commits/1adccd8f4d9a16f0c94417cadcc4928d49a1a8e9